### ISSUE TITLE
[ortmodule] GraphExecutionManager self._device = None

### ISSUE THREAD

**Issue by jiafatom** [microsoft/onnxruntime#10740]
**Describe the bug**
I want to wrap a single Dropout module into ORTModule: `self.pos_drop = nn.Dropout(p=drop_rate)`
However, it fails because this module has no parameters, then 
self._device = _utils.get_device_from_module(module) is None
There is check in `def _get_session_config(self):`
`if self._device.type == 'cuda':` which will crash because `self._device is None`.
It seems that we should automatically set `self._device` to an available EP like cuda if self._device is None.

**Comment by jiafatom** [comment]
cc @baijumeswani, @ashbhandare

**Comment by thiagocrepaldi** [comment]
Dropout is not a trainable oeprator. Does it make sense to use a non trainable module in ortmodule? Maybe we should just improve error message

**Comment by jiafatom** [comment]
@thiagocrepaldi the background is that I have a big model, and I want to wrap part of the model using `ORTModule`. I just wrap several submodules, and then the forward pass crashes. Then it takes some time to find the reason because I don't know one submodule is a non-trainable model. So for non-trainable model, is there a way to make it pass without crashing?
